### DISCUSSION TITLE
PrimeVue.org site not remembering theme issue.

### DISCUSSION THREAD

**Discussion by MPGod42** [primefaces/community#2941] [PrimeVue]
I'm using the primevue.org site to read documentation and any time the site is opened or refreshed, i get flashbanged by the light mode.

Tried pc, mobile, different browsers and all have the issue.

**Comment by mergehez** [comment]
the issue was "temporarily" closed because they think it is not a "bug": https://github.com/primefaces/primevue/issues/6461
the PR also was rejected: https://github.com/primefaces/primevue/pull/6462
